src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -268,6 +272,16 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 3,
                 "The request body size limit has been disabled.");
+
+            _cannotApplyRequestFormLimits = LoggerMessage.Define(
+                LogLevel.Warning,
+                1,
+                "Could not apply limits to the request form as it might have been read already.");
+
+            _appliedRequestFormLimits = LoggerMessage.Define(
+                LogLevel.Debug,
+                2,
+                "Applied limits to the request form.");
```

Comment:
Any suggestions regarding the log message here?

Reply:
> Read request form with the configured form options.

Reply:
This doesn't actually do reading though